src/lcode.c

```diff
@@ -382,11 +386,23 @@ static void removelastinstruction (FuncState *fs) {
 int luaK_code (FuncState *fs, Instruction i) {
   Proto *f = fs->f;
   /* put new instruction in code array */
+  int idx = fs->pc;
   luaM_growvector(fs->ls->L, f->code, fs->pc, f->sizecode, Instruction,
                   MAX_INT, "opcodes");
   f->code[fs->pc++] = i;
+#ifdef WITH_YK
+  // YKOPT: Use doubling allocation or similar.
```

Comment:
"doubling allocation"?

Reply:
Over-allocate twice as much as you did before.

Reply:
It would be clearer, IMHO, to say something like "Reallocating on every new opcode is inefficient."

Reply:
Fixed in a4c5fd10a68